distributed/scheduler.py

```diff
@@ -2621,8 +2630,8 @@ def _transition_processing_erred(
         self,
         key: Key,
         stimulus_id: str,
+        worker: str | None = None,
```

Comment:
How can a task be processing without a worker? Is it when the worker it was processing on died and it caused the task to increase its suspicious count too much? If so, it may be a good idea to note it in a comment?